orchard/orchard_runner.go

```diff
@@ -94,25 +94,30 @@ func cmdOutput(cmd *exec.Cmd) ([]byte, []byte, error) {
 	return b.Bytes(), c.Bytes(), err
 }
 
-func processCmd(command string, pwd string) (string, error) {
+func processCmd(command string, pwd string) ([]string, error) {
 	if err := os.Chdir(pwd); err != nil {
-		return "", fmt.Errorf("cd %v has error: %w", pwd, err)
+		return []string{}, fmt.Errorf("cd %v has error: %w", pwd, err)
 	}
 	cmds, err := parseCommandLine(command)
 	if err != nil {
-		return "", fmt.Errorf("parse command line error %w", err)
+		return []string{}, fmt.Errorf("parse command line error %w", err)
 	}
 	if len(cmds) < 1 {
-		return "", fmt.Errorf("Invalid command line %s", command)
+		return []string{}, fmt.Errorf("invalid command line %s", command)
 	}
 	cmd := exec.Command(cmds[0], cmds[1:]...)
 	stdout, stderr, err := cmdOutput(cmd)
 	output := string(stdout)
 	if err != nil {
 		combinedOutput := fmt.Sprintf("%s\n%s", output, string(stderr))
-		return "", fmt.Errorf("exec command %v has error: %w: %s", command, err, combinedOutput)
+		return []string{}, fmt.Errorf("exec command %v has error: %w: %s", command, err, combinedOutput)
 	}
-	return output, nil
+	var outputs []string
+	err = json.Unmarshal([]byte(output), &outputs)
+	if err != nil {
+		return []string{}, fmt.Errorf("marshall error %w", err)
```

Comment:
let's just wrap this with an array and return the original content, so that it's backwards compatible

Reply:
@realstraw still return the original `output`? so `return output, fmt.Errorf("marshall error %w", err)`?

Reply:
@realstraw gotcha, wrap if error to make it backward compatible